examples/fluids/qfunctions/blasius.h

```diff
@@ -50,34 +55,32 @@ CEED_QFUNCTION(ICsBlasius)(void *ctx, CeedInt Q,
   CeedScalar (*q0)[CEED_Q_VLA] = (CeedScalar(*)[CEED_Q_VLA])out[0];
 
   const BlasiusContext context = (BlasiusContext)ctx;
-  const CeedScalar cv     = context->newtonian_ctx.cv;
-  const CeedScalar cp     = context->newtonian_ctx.cp;
-  const CeedScalar gamma  = cp / cv;
-  const CeedScalar mu     = context->newtonian_ctx.mu;
-
-  const CeedScalar theta0 = context->theta0;
-  const CeedScalar P0     = context->P0;
-  const CeedScalar delta0 = context->delta0;
-  const CeedScalar Uinf   = context->Uinf;
-  const CeedScalar x_inflow   = context->x_inflow;
-
-  const CeedScalar e_internal = cv * theta0;
-  const CeedScalar rho        = P0 / ((gamma - 1) * e_internal);
-  const CeedScalar x0         = Uinf*rho / (mu*25/ (delta0*delta0) );
-  CeedScalar u, v, T, t12;
+  const CeedScalar cv          = context->newtonian_ctx.cv;
+  const CeedScalar cp          = context->newtonian_ctx.cp;
+  const CeedScalar gamma       = cp / cv;
+  const CeedScalar mu          = context->newtonian_ctx.mu;
+  const CeedScalar theta0      = context->theta0;
+  const CeedScalar P0          = context->P0;
+  const CeedScalar delta0      = context->delta0;
+  const CeedScalar Uinf        = context->Uinf;
+  const CeedScalar x_inflow    = context->x_inflow;
+  const CeedScalar e_internal  = cv * theta0;
+  const CeedScalar rho         = P0 / ((gamma - 1) * e_internal);
+  const CeedScalar x0          = Uinf*rho / (mu*25/(delta0*delta0));
+  CeedScalar t12;
 
   // Quadrature Point Loop
   CeedPragmaSIMD
   for (CeedInt i=0; i<Q; i++) {
-    const CeedScalar x[] = {X[0][i], X[1][i], X[2][i]};
+    const CeedScalar x[3] = {X[0][i], X[1][i], 0.};
+    State s = BlasiusSolution(context, x, x0, x_inflow, rho, &t12);
+    CeedScalar q[5] = {0};
+    UnpackState_U(s.U, q);
+    for (CeedInt j=0; j<5; j++) q0[j][i] = q[j];
 
-    BlasiusSolution(context, x[1], x0, x[0] - x_inflow, rho, &u, &v, &T, &t12);
+    // ToDo: Need to drop this line
+    q0[4][i] = rho * (e_internal + 0.5*Dot3(s.Y.velocity, s.Y.velocity));
```

Comment:
It looks like `rho` and `e_internal` here are constant. How different are they from the computed values? (Maybe set a breakpoint here to inspect in the debugger, or use printf and filter the output.)

Reply:
There was a memory issue that is fixed now. It diverges much later (`t=0.000286`) now. The major difference I am seeing now is that the `f[]` values are much larger than `f`, `fp`, `fpp` on main (incompressible Blasius) which makes the values of u, v, and t12 ~22-27 times bigger. The value of `h[0]` and the computed temperature makes sense though!

Reply:
So we don't match `Uinf` as specified? How does it compare if you spot-check it versus the compressible Julia code? We spot-checked the PETSc example so it should agree with the Julia, which visually matched diagrams from the literature.

Reply:
I spot-checked it versus the Julia code and they were identical. I had to increase the value of `delta0` from `4.2e-4` to `4.2e-3` to have $\eta_{max} = 5$ which helped with converging. 
![Screenshot from 2022-08-10 12-31-59](https://user-images.githubusercontent.com/49916147/183991226-947a1204-4a37-415d-837f-27fb73a19bb1.png)
These are taken after 40 time-steps. The left plot is the compressible blasius solution (this branch) and the right one is run on main.

Reply:
Can you make a line chart at the inflow?

Reply:
And with the smaller `delta0`, I'm assuming it struggled to converge because $`\eta_{\max} = 50`$? I think in practice, the (laminar) far field is constant (to fairly strict tolerance, though we could quantify) beyond about $`\eta_{\max} = 10`$. So we could solve on that limited domain and extrapolate. Or we can pick a better transform from $`[0,\infty) \to [-1, 1]`$.

Reply:
That's right, $\eta_{max}$ was 50.

Reply:
At Ma = 0:
![Screenshot from 2022-08-25 14-44-36](https://user-images.githubusercontent.com/49916147/186770505-2aa83dd0-b98a-45ae-81c6-a39ea6f77dbd.png)